MIGRATING.MD

```diff
@@ -3,20 +3,22 @@
 The Splunk Distribution of OpenTelemetry Java Instrumentation replaces the
 [SignalFx Java Agent](https://github.com/signalfx/signalfx-java-tracing).
 If you're using the SignalFx Java Agent, migrate to the Splunk Distribution of
-OpenTelemetry Java Instrumentation to use OpenTelemetry's instrumentation to
-send traces to Splunk APM. The Splunk Distribution of OpenTelemetry Java
-Instrumentation uses OpenTelemetry to instrument applications, which is an
-open-source API to gather telemetry data, and has a smaller footprint.
+OpenTelemetry Java Instrumentation.
+
+The distribution uses [OpenTelemetry Java Instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation) 
+to instrument applications, which in open-source, robust and has a smaller memory footprint.
+
+## Tracing data
```

Comment:
Hmm, how about `Telemetry changes`?

Reply:
The idea is to have a separate section describing the data (perhaps extended with a table comparing main differences). But will change to "Telemetry data" as it sounds better.